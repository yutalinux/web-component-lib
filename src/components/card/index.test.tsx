import { expect, test } from "bun:test";
import { createElement } from "react";
import { renderToString } from "react-dom/server";
import Card from ".";

test("Button Snapshot", () => {
  const button = createElement(Card, null)
  const renderedString = renderToString(button)
  expect(renderedString).toMatchSnapshot()
})
