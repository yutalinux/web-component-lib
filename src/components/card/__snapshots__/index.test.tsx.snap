// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Snapshot 1`] = `"<div class="bg-white text-black border p-2 rounded-md min-h-32"></div>"`;
